test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -908,6 +908,57 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 				},
 			},
 		),
+		Entry("[test_id:54021][tier1][testtype4][tmscope:container][devices] pod with two gu cnt land on a node with enough resources, containers should be spread on a different zone",
+			nrtv1alpha1.SingleNUMANodeContainerLevel,
+			setupPaddingContainerLevel,
+			e2enrt.CheckNodeConsumedResourcesAtLeast,
+			podResourcesRequest{
+				appCnt: []corev1.ResourceList{
+					{
+						corev1.ResourceCPU:    resource.MustParse("4"),
+						corev1.ResourceMemory: resource.MustParse("4Gi"),
+						"example.com/deviceA": resource.MustParse("2"),
+						"example.com/deviceC": resource.MustParse("1"),
```

Comment:
this test will be executed only if those two devices are on the cluster, mentioning these explicitly here will work for current sample devices. we might need to tweak those instances in the future when supporting real devices on the cluster, and have the names of the devices based on environment variables so the test works for any kind of device (real/sample). 
I've started working on this, but still needs some adjustments and testing:
https://github.com/shajmakh/numaresources-operator/commit/64c25286110a19fb03ea35e44fc49d214a807bea

Reply:
right, I overlooked this part

Reply:
Just make sure to update the variables here as well:
https://github.com/openshift-kni/numaresources-operator/tree/main/test/e2e/serial#e2e-serial-testsuite